internal/host/transition_test.go

```diff
@@ -1017,6 +1017,61 @@ var _ = Describe("Refresh Host", func() {
 		hostId = strfmt.UUID(uuid.New().String())
 		clusterId = strfmt.UUID(uuid.New().String())
 	})
+
+	Context("host installation timeout", func() {
```

Comment:
please add a test for a stage that is not in the map and verify that it stays in the correct status and another test that verifies that if the time didn't get to the threshold - it stays in the correct status